src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -2317,16 +2317,14 @@ components:
 
     TwoWayLinkedDappsCollection:
       description: A collection of two-way linked (resolved & verified) dApps linked to the entity defining this collection.
-      allOf:
-        - $ref: "#/components/schemas/ResultSetCursorMixin"
-        - type: object
-          required:
-            - items
-          properties:
-            items:
-              type: array
-              items:
-                $ref: "#/components/schemas/TwoWayLinkedDappsCollectionItem"
+      type: object
```

Comment:
I'd have thought removing `ResultSetCursorMixin` is a breaking schema change?

I think we may be forced to keep it sadly, else people using a modern SDK might be broken when trying to read these endpoints after its removal.

Reply:
yes they were optional and we've never returned them.. there's no endpoint to paginate over that :D